[Infinitest for Eclipse] Plugin not working with JDK16

**Context**
 - Plugin: Infinitest for Eclipse
 - Plugin Version: 5.2.0
 - IDE: Eclipse 2021-09 (4.21.0)

**Describe the bug**
- The infinitest status bar (bottom left) is not there
- The infinitest settings section has an error

**To Reproduce**
- Install Eclipse 2021-09
- Install infinitest through the marketplace
- Restart Eclipse
- Error, see below

**Expected behavior**
- Eclipse 2021-09 should be supported

**Screenshots**
![image](https://user-images.githubusercontent.com/3357406/134784714-287d6ffc-11e0-42d8-819e-29f6355ccab0.png)

```
eclipse.buildId=4.21.0.I20210906-0500
java.version=16.0.2
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=fr_CA
Framework arguments:  -product org.eclipse.epp.package.java.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.java.product

org.eclipse.ui
Error
Sat Sep 25 15:59:00 EDT 2021
Unable to execute early startup code for the org.eclipse.ui.IStartup extension contributed by the 'org.infinitest.eclipse' plug-in.

java.lang.IllegalStateException: Cannot load configuration class: org.infinitest.eclipse.InfinitestConfig
	at org.springframework.context.annotation.ConfigurationClassPostProcessor.enhanceConfigurationClasses(ConfigurationClassPostProcessor.java:403)
	at org.springframework.context.annotation.ConfigurationClassPostProcessor.postProcessBeanFactory(ConfigurationClassPostProcessor.java:249)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:281)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:125)
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:687)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:525)
	at org.springframework.context.annotation.AnnotationConfigApplicationContext.<init>(AnnotationConfigApplicationContext.java:84)
	at org.infinitest.eclipse.InfinitestPlugin.getBean(InfinitestPlugin.java:119)
	at org.infinitest.eclipse.InfinitestPlugin.getPluginController(InfinitestPlugin.java:114)
	at org.infinitest.eclipse.InfinitestPlugin.startContinuouslyTesting(InfinitestPlugin.java:92)
	at org.infinitest.eclipse.ContinuousTestingStarter.initializeAutoTesting(ContinuousTestingStarter.java:48)
	at org.infinitest.eclipse.ContinuousTestingStarter.earlyStartup(ContinuousTestingStarter.java:41)
	at org.eclipse.ui.internal.EarlyStartupRunnable.runEarlyStartup(EarlyStartupRunnable.java:79)
	at org.eclipse.ui.internal.EarlyStartupRunnable.run(EarlyStartupRunnable.java:55)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.ui.internal.Workbench$40.run(Workbench.java:2667)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.springframework.cglib.proxy.Enhancer
	at org.springframework.context.annotation.ConfigurationClassEnhancer.newEnhancer(ConfigurationClassEnhancer.java:122)
	at org.springframework.context.annotation.ConfigurationClassEnhancer.enhance(ConfigurationClassEnhancer.java:110)
	at org.springframework.context.annotation.ConfigurationClassPostProcessor.enhanceConfigurationClasses(ConfigurationClassPostProcessor.java:393)
	... 16 more
```

--- a/infinitest-eclipse/pom.xml
+++ b/infinitest-eclipse/pom.xml
@@ -288,12 +288,12 @@
 			<artifactId>infinitest-lib</artifactId>
 			<version>${project.parent.version}</version>
 		</dependency>
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-context</artifactId>
-      <version>4.3.9.RELEASE</version>
-    </dependency>
-    <dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-context</artifactId>
+			<version>5.3.22</version>
+		</dependency>
+		<dependency>
 			<groupId>org.eclipse</groupId>
 			<artifactId>core.net</artifactId>
 			<version>1.0.0.I20070531</version>

